Code/GraphMol/Fingerprints/test1.cpp

```diff
@@ -3178,6 +3179,116 @@ void testGitHubIssue811() {
   BOOST_LOG(rdErrorLog) << "  done" << std::endl;
 }
 
+void testRDKFPUnfolded(){
```

Comment:
I'd like to see this test, or at least part of it, duplicated for the python wrappers as well.
It's easier to write these in Python. :-)